+ source ../common_lib/fsio_lib.sh
++ source /root/mugen/libs/locallibs/common_lib.sh
+++ python3 --version
+++ '[' 0 -eq 0 ']'
+++ source /root/mugen/libs/locallibs/common_lib_python.sh
++ export 'FS_TYPE=ext3 ext4 xfs'
++ FS_TYPE='ext3 ext4 xfs'
++ export LANG=en_US.UTF-8
++ LANG=en_US.UTF-8
++ REMOVE_VG
++ vggroup=
++ [[ '' != '' ]]
+++ df -T
+++ grep test_vggroup
+++ awk '{print $7}'
++ mounted=
++ [[ '' != '' ]]
++ lsblk
++ grep test_vggroup
++ [[ 1 -eq 0 ]]
++ rm -rf /tmp/point120230428212956 /tmp/point220230428212956 /tmp/point320230428212956
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start environment preparation.'
+ message='Start environment preparation.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start environment preparation.'
Fri Apr 28 21:30:18 2023 - INFO  - Start environment preparation.
+ point_list=($(CREATE_FS "ext3 ext4 xfs"))
++ CREATE_FS 'ext3 ext4 xfs'
++ fs_type='ext3 ext4 xfs'
+++ date +%Y%m%d%H%M%S
++ cur_date=20230428213018
+++ CREATE_VG
++++ date +%Y%m%d%H%M%S
+++ cur_date=20230428213018
++++ lsblk
++++ grep disk
++++ awk '{print $1}'
++++ tail -n 1
+++ free_disk=vdt
+++ disk_name=/dev/vdt
+++ pvcreate /dev/vdt -f
+++ [[ 5 -eq 0 ]]
++++ pvcreate /dev/vdt
++++ grep test_vggroup
++++ cut -d '"' -f 4
+++ vggroup=test_vggroup20230428204559
+++ printf test_vggroup20230428204559
++ vggroup=test_vggroup20230428204559
++ count=1
++ msg='test_vggroup20230428204559 '
++ for fs in ${fs_type[@]}
++ lvname=test_lv120230428213018
++ point=/tmp/point120230428213018
++ lvcreate -n test_lv120230428213018 -L 512M test_vggroup20230428204559 -y
  Volume group "test_vggroup20230428204559" has insufficient free space (124 extents): 128 required.
++ mkfs -t ext3 /dev/test_vggroup20230428204559/test_lv120230428213018
mke2fs 1.46.5 (30-Dec-2021)
The file /dev/test_vggroup20230428204559/test_lv120230428213018 does not exist and no size was specified.
++ mkdir /tmp/point120230428213018
++ mount /dev/test_vggroup20230428204559/test_lv120230428213018 /tmp/point120230428213018
mount: /tmp/point120230428213018: special device /dev/test_vggroup20230428204559/test_lv120230428213018 does not exist.
++ msg='test_vggroup20230428204559 /tmp/point120230428213018 '
++ count=2
++ for fs in ${fs_type[@]}
++ lvname=test_lv220230428213018
++ point=/tmp/point220230428213018
++ lvcreate -n test_lv220230428213018 -L 512M test_vggroup20230428204559 -y
  Volume group "test_vggroup20230428204559" has insufficient free space (124 extents): 128 required.
++ mkfs -t ext4 /dev/test_vggroup20230428204559/test_lv220230428213018
mke2fs 1.46.5 (30-Dec-2021)
The file /dev/test_vggroup20230428204559/test_lv220230428213018 does not exist and no size was specified.
++ mkdir /tmp/point220230428213018
++ mount /dev/test_vggroup20230428204559/test_lv220230428213018 /tmp/point220230428213018
mount: /tmp/point220230428213018: special device /dev/test_vggroup20230428204559/test_lv220230428213018 does not exist.
++ msg='test_vggroup20230428204559 /tmp/point120230428213018 /tmp/point220230428213018 '
++ count=3
++ for fs in ${fs_type[@]}
++ lvname=test_lv320230428213018
++ point=/tmp/point320230428213018
++ lvcreate -n test_lv320230428213018 -L 512M test_vggroup20230428204559 -y
  Volume group "test_vggroup20230428204559" has insufficient free space (124 extents): 128 required.
++ mkfs -t xfs /dev/test_vggroup20230428204559/test_lv320230428213018
mkfs: failed to execute mkfs.xfs: No such file or directory
++ mkdir /tmp/point320230428213018
++ mount /dev/test_vggroup20230428204559/test_lv320230428213018 /tmp/point320230428213018
mount: /tmp/point320230428213018: special device /dev/test_vggroup20230428204559/test_lv320230428213018 does not exist.
++ msg='test_vggroup20230428204559 /tmp/point120230428213018 /tmp/point220230428213018 /tmp/point320230428213018 '
++ count=4
++ echo test_vggroup20230428204559 /tmp/point120230428213018 /tmp/point220230428213018 /tmp/point320230428213018
+ ext3_point=/tmp/point120230428213018
+ ext4_point=/tmp/point220230428213018
+ xfs_point=/tmp/point320230428213018
+ echo 'test ext3 file'
+ LOG_INFO 'End of environmental preparation!'
+ message='End of environmental preparation!'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End of environmental preparation!'
Fri Apr 28 21:30:22 2023 - INFO  - End of environmental preparation!
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Fri Apr 28 21:30:23 2023 - INFO  - Start to run test.
+ mv /tmp/point120230428213018/testFile1 /tmp/point220230428213018/testFile2
+ stat /tmp/point220230428213018/testFile2
+ grep Inode
Device: 0,38	Inode: 3290        Links: 1
+ CHECK_RESULT 0 0 0 'Check inode of file /tmp/point220230428213018/testFile2 failed.'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Check inode of file /tmp/point220230428213018/testFile2 failed.'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ grep test /tmp/point220230428213018/testFile2
test ext3 file
+ CHECK_RESULT 0 0 0 'Check file msg of /tmp/point220230428213018/testFile2 failed.'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Check file msg of /tmp/point220230428213018/testFile2 failed.'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ mv /tmp/point220230428213018/testFile2 /tmp/point320230428213018/testFile3
+ stat /tmp/point320230428213018/testFile3
+ grep Inode
Device: 0,38	Inode: 3290        Links: 1
+ CHECK_RESULT 0 0 0 'Check inode of file /tmp/point320230428213018/testFile3 failed.'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Check inode of file /tmp/point320230428213018/testFile3 failed.'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ grep test /tmp/point320230428213018/testFile3
test ext3 file
+ CHECK_RESULT 0 0 0 'Check file msg of /tmp/point320230428213018/testFile3 failed.'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Check file msg of /tmp/point320230428213018/testFile3 failed.'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ mv /tmp/point320230428213018/testFile3 /tmp/point120230428213018/testFile4
+ stat /tmp/point120230428213018/testFile4
+ grep Inode
Device: 0,38	Inode: 3290        Links: 1
+ CHECK_RESULT 0 0 0 'Check inode of file /tmp/point120230428213018/testFile4 failed.'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Check inode of file /tmp/point120230428213018/testFile4 failed.'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ grep test /tmp/point120230428213018/testFile4
test ext3 file
+ CHECK_RESULT 0 0 0 'Check file msg of /tmp/point120230428213018/testFile4 failed.'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Check file msg of /tmp/point120230428213018/testFile4 failed.'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Fri Apr 28 21:30:24 2023 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z ''
+ test 0 -eq 0
+ LOG_INFO 'succeed to execute the case.'
+ message='succeed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'succeed to execute the case.'
Fri Apr 28 21:30:25 2023 - INFO  - succeed to execute the case.
+ exec_result=
+ exit 0
+ post_test
+ LOG_INFO 'Start to restore the test environment.'
+ message='Start to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to restore the test environment.'
Fri Apr 28 21:30:26 2023 - INFO  - Start to restore the test environment.
++ echo test_vggroup20230428204559 /tmp/point120230428213018 /tmp/point220230428213018 /tmp/point320230428213018
+ list='test_vggroup20230428204559 /tmp/point120230428213018 /tmp/point220230428213018 /tmp/point320230428213018'
+ REMOVE_FS 'test_vggroup20230428204559 /tmp/point120230428213018 /tmp/point220230428213018 /tmp/point320230428213018'
+ point_list=($1)
++ seq 0 3
+ for i in $(seq 0 $((${#point_list[@]} - 1)))
+ tmp=test_vggroup20230428204559
++ df -T
++ grep test_vggroup20230428204559
++ awk '{print $1}'
+ lv=
+ [[ '' == '' ]]
+ continue
+ for i in $(seq 0 $((${#point_list[@]} - 1)))
+ tmp=/tmp/point120230428213018
++ df -T
++ grep /tmp/point120230428213018
++ awk '{print $1}'
+ lv=
+ [[ '' == '' ]]
+ continue
+ for i in $(seq 0 $((${#point_list[@]} - 1)))
+ tmp=/tmp/point220230428213018
++ df -T
++ grep /tmp/point220230428213018
++ awk '{print $1}'
+ lv=
+ [[ '' == '' ]]
+ continue
+ for i in $(seq 0 $((${#point_list[@]} - 1)))
+ tmp=/tmp/point320230428213018
++ df -T
++ grep /tmp/point320230428213018
++ awk '{print $1}'
+ lv=
+ [[ '' == '' ]]
+ continue
+ LOG_INFO 'End to restore the test environment.'
+ message='End to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to restore the test environment.'
Fri Apr 28 21:30:27 2023 - INFO  - End to restore the test environment.
